core/vm/evm.go

```diff
@@ -34,19 +34,23 @@ var emptyCodeHash = crypto.Keccak256Hash(nil)
 
 type (
 	// CanTransferFunc is the signature of a transfer guard function
-	CanTransferFunc   func(StateDB, common.Address, *big.Int) bool
+	CanTransferFunc func(StateDB, common.Address, *big.Int) bool
+	// CanTransferMCFunc is the signature of a transfer native asset function
```

Comment:
// Return value 0 indicates the address can transfer the given amount of the MultiCoin
// Return value 1 indicates that it may not.